content/v2/openapi.yml

```diff
@@ -4903,6 +4929,29 @@ components:
               - ns2.dnsimple.com
               - ns3.dnsimple.com
               - ns4.dnsimple.com
+    ZoneNsRecordsUpdate:
+      description: Zone name server attributes
+      content:
+        application/json:
+          schema:
+            type: object
+            properties:
+              ns_names:
+                type: array
+                items:
+                  type: string
+                example:
+                  - ns1.dnsimple.com
+                  - ns2.dnsimple.com
+                  - ns3.dnsimple.com
+                  - ns4.dnsimple.com
+              ns_set_ids:
```

Comment:
For clarification: Is it correct that NS Sets will only be created through the UI (at least for now)?

Reply:
Correct! There are no endpoints to manage name server sets.

We'll have to see how sets are used in the hands of customers, but my bet is that people will not have an API-requiring number of sets. Of course, now that I've said that, I'm sure that'll be the first feature request...